### PR TITLE
Added allowedAttributes to Editable Action

### PR DESCRIPTION
**Pull request by sebathi**
Some times you use it as a fast edit and don't want to have all items to be updated. only the ones you are changing. It is faster than creating a scenario.

### CODE REVIEW

File: src/EditableAction.php [hunk 1]
```diff
@@ -68,7 +68,7 @@ public function run()
         if ($value === null) {
             throw new BadRequestHttpException("'value' parameter cannot be empty.");
         }
-        /** @var \yii\db\ActiveRecord $model */
+        /** @var \Yii\db\ActiveRecord $model */
```
Comment (tonydspaniard) [inline src/EditableAction.php]: Doesn't seem correct namespace

File: src/EditableAction.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * @copyright Copyright (c) 2013-2015 2amigOS! Consulting Group LLC
+ * @copyright Copyright (c) 2013 2amigOS! Consulting Group LLC
```
Comment (tonydspaniard) [inline src/EditableAction.php]: Should be updated to 20013-2016 instead? :)

**Comment by tonydspaniard** [comment]
@sebathi thanks... sample usage and result?

**Comment by sebathi** [comment]
It will only let you update the attributes you add to the variable. for example:

'action'=>[
'class' EditableAction::className(),
'allowedAttributes'=>['userId', 'notes', 'stateId'],
....
]

That will let you only update these three attributes of the model.

**Comment by sebathi** [comment]
?

**Comment by sebathi** [comment]
??

**Comment by sebathi** [comment]
Great!!

**Comment by sebathi** [comment]
Can you create a version? I can remove my fork on my project now ;)

**Comment by tonydspaniard** [comment]
@sebathi I did. I am very sorry for the delay. Is version 1.0.0

**Comment by tonydspaniard** [comment]
Wasn't able to fully test it with latest Yii2 changes. Please, let me know if all works correctly Thanks

**Pull request merged**